src/main.ts

```diff
@@ -50,9 +52,13 @@ async function bootstrap(): Promise<void> {
     const document = SwaggerModule.createDocument(app, options);
     SwaggerModule.setup('/', app, document);
 
-    // await app.startAllMicroservicesAsync();
+    // Connects to Rabbit etc
+    await app.startAllMicroservicesAsync();
+
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
```

Comment:
I think we should change back to the eslint configuration of nestjs as there are more of these caveats that they already resolved.

Reply:
Such as?